### DISCUSSION TITLE
What do you like to see cats doing?

### DISCUSSION THREAD

**Discussion by yuzu-ginger** [NAIST-SE/AttractingContributors#1070] [2024 Attraction Contributions]
I have created a new repository called [Ilovecats](https://github.com/yuzu-ginger/Ilovecats).
The project aims to share everything about cats, including interesting facts and personal stories.

I have a black cat. She is very cute! Do you have cats? or go to cat cafes?

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
